src/Compilers/CSharp/csc/Program.cs

```diff
@@ -12,7 +12,11 @@ public static int Main(string[] args)
             => Main(args, SpecializedCollections.EmptyArray<string>());
 
         public static int Main(string[] args, string[] extraArgs)
-            => DesktopBuildClient.Run(args, extraArgs, RequestLanguage.CSharpCompile, Csc.Run, new SimpleAnalyzerAssemblyLoader());
+        {
+            var result = DesktopBuildClient.Run(args, extraArgs, RequestLanguage.CSharpCompile, Csc.Run, new SimpleAnalyzerAssemblyLoader());
+            if (System.Diagnostics.Debugger.IsAttached) System.Console.ReadLine();
+            return result;
+        }
```

Comment:
Accident. I want the csc window to stay open after a run, to inspect the logged errors. Will revert it with the next push.